build.sh

```diff
@@ -18,8 +18,8 @@
 
 WORKSPACE_DIR=
 THREADS=1
-CLEAN=false
-TEST=false
+CLEAN=true
+TEST=true
```

Comment:
These are the default values; you shouldn't be changing them. If you want to run tests or do a clean build pass the `-c` or `-t` flags from the command line.